osmoutils/slice_helper_test.go

```diff
@@ -0,0 +1,28 @@
+package osmoutils
```

Comment:
```suggestion
package osmoutils_test
```

Reply:
When I make this change, I get the `undeclared name: ReverseSlice` error. Looking at `binary_search_test.go`, it also has package set to `osmoutils`. Have I configured something incorrectly?

Reply:
You can fix this by importing osmoutils like below. I'm not sure why we dont use package osmoutils_test in `binary_search_test.go`. Maybe it's worth changing both or keeping this package name the same as `binary_search_test.go` 

```
import (
	"github.com/osmosis-labs/osmosis/v10/osmoutils"
...
)

osmoutils.ReverseSlice(...)
```

Reply:
Thanks for the tip sir. Still getting used to Go 😅